Update gmf layertree icons

ger-benjamin opened the pull request:
Fix: https://github.com/camptocamp/ngeo/issues/644

Example: https://ger-benjamin.github.io/ngeo/gmf_layertree_icons/examples/contribs/gmf/apps/mobile

If we want that this example matches exactly with the example in the design document, I must create a new, plain, "node group" icon. For now, I just play with color's brightness.

pgiraud commented on contribs/gmf/less/layertree.less:
```diff
@@ -13,29 +13,30 @@ gmf-layertree ul .treenode {
     }
     &.indeterminate::after {
-      content: "\f147";
+      color: darken(@brand-primary, 25%);
```
Can you have a try with `hsv(hue(@brand-primary), 40%, 60%)`?

ger-benjamin commented:
@pgiraud : 
 - I've moved the "on/off/indeterminate" class on the node (not only on the "state")
 - I've set the opacity off "off-state-nodes"  to 0.5
 - I've changed the indeterminate visual effect to the "special-link" color (please, help me to find a better name...)

ger-benjamin commented:
(note to myself, example is probably visually broken)

sbrunner commented on contribs/gmf/less/vars.less:
```diff
@@ -9,6 +9,7 @@
 @map-tools-size: 29px;
 @onhover-color: darken(@nav-bg, 20%);
 @search-result-font-color: black;
+@special-link: hsv(hue(@brand-primary), 40%, 60%);
```
looks strange ..., shouldn't be `hsv(hue(@brand-primary, 40%), 60%)`?

pgiraud replied:
Nope, this is correct. `hsv` takes 3 arguments, `hue` takes one.

sbrunner replied:
Effectively, your right :-), sorry...

ger-benjamin commented:
Done. Please review (beware of your cache with the example)

pgiraud commented:
This looks good to me.
Let's leave the name `special-link` as is for now.
We'll need to do some user tests and figure out if people understand what happens.
